### PR TITLE
Upload changes

### PR DESCRIPTION
**Pull request by agnes-kiss**
Changed the zip method to S3 for the upload files and added BigQuery and Databricks support.

### CODE REVIEW

File: content/upload/images/sql_workspace.png
Comment (rlh1994) [inline content/upload/images/sql_workspace.png:1]: I think this is technically the old UI?

File: content/_index.md
Comment (rlh1994) [inline content/_index.md:1]: Maybe should get rid of the banner image at the top now?

File: content/_index.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ end
 
 subgraph cloud[Cloud]
     snowplow[Snowplow BDP/OS Cloud]
-    snowflake[(Snowflake)]
+    warehouse[(Warehouse)]
```
Comment (rlh1994) [inline content/_index.md]: I feel like the streamlit part should have the arrow go _out_ of the warehouse? Also snowflake seems to be used elsewhere and the warehouse just sort of sat on the side by itself?

File: content/_index.md [hunk 2]
```diff
@@ -77,7 +77,7 @@ tracker -- "Sends tracked events" --> snowplow
 
 In approximately 10 working hours you can achieve the following:
 
-- **Upload data -** Upload a sample Snowplow events dataset to your Snowflake warehouse
+- **Upload data -** Upload a sample Snowplow events dataset to your warehouse
```
Comment (rlh1994) [inline content/_index.md:35]: Per John's point, can we maybe be optimistic with these times now?

File: content/_index.md
Comment (rlh1994) [reply content/_index.md:1]: I think also move the system overview to the bottom of the page, it's more important to know who it's for and the prereqs FIRST

File: content/upload/1-upload.md [hunk 3]
```diff
@@ -411,18 +331,352 @@ SELECT
...
+#### **Step 2:**  Upload the csv file
+
+2.1 Open the `BigQuery` page in the [Google Cloud console](https://console.cloud.google.com/).
+
+2.2 In the `Explorer` panel, expand your project and select the `atomic` schema/dataset. You should see the details panel open, if not, expand the `Actions` option (click on the three vertical dots) and click `Open`.
+
+2.3 In the `Details` panel, click `Create table`.
+
+3.4 On the `Create table` page fill out the sections:
+
```
Comment (rlh1994) [inline content/upload/1-upload.md]: 3.4?!?!?!?!?!?!?!?!?!?

**Review by rlh1994 — CHANGES_REQUESTED** [review]
1 typo and then just some ideas you can choose to ignore

File: content/upload/1-upload.md (hunk 3 above)
Comment (agnes-kiss) [reply content/upload/1-upload.md]: image

Comment (rlh1994) [reply content/upload/1-upload.md]: Yeah 2.3 is right, it's the next number that's wrong

Comment (agnes-kiss) [reply content/upload/1-upload.md]: The point was that it needs correcting in two more accelerators 😄

File: content/_index.md (hunk 2 above)
Comment (agnes-kiss) [reply content/_index.md:35]: I changed the visualise from 3 hours to 2 but even this is too optimistic considering setting up the env will take quite a while.

File: content/_index.md (hunk 1 above)
Comment (agnes-kiss) [reply content/_index.md]: true, got this fixed

**Review by rlh1994 — APPROVED** [review]
LGTM! Just remember to add the new warehouses to the dashboard in the shared resources repo as well.

**Pull request merged**